lucien-ios/Models/Dashboard.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MyCollection.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/5/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+struct Dashboard: Decodable {
+    var myCollectionCount: Int
+
+    enum CodingKeys: String, CodingKey {
+        case dashboard
+    }
+
+    enum DashboardKeys: String, CodingKey {
+        case myCollectionCount = "my_collection_count_by_issue"
+    }
+
+    init(from decoder: Decoder) throws {
```

Comment:
This is fine for now, but we can potentially make this a `DashboardResponse` class that decodes a `Dashboard` in the same way we do with `User`. As long as `Dashboard` is defined outside of `DashboardResponse` we'll be okay

Reply:
In my `gracie/dashboard-lending` that's the way I have it!

Reply:
Perfect, you can leave it for this PR then